srtcore/api.cpp

```diff
@@ -433,236 +441,240 @@ SRTSOCKET CUDTUnited::newSocket()
    if (!ns)
       throw CUDTException(MJ_SYSTEMRES, MN_MEMORY, 0);
 
+    if (pps)
+        *pps = ns;
+
    return ns->m_SocketID;
 }
 
 int CUDTUnited::newConnection(const SRTSOCKET listen, const sockaddr_any& peer, const CPacket& hspkt,
         CHandShake& w_hs, SRT_REJECT_REASON& w_error)
 {
-   CUDTSocket* ns = NULL;
+    CUDTSocket* ns = NULL;
 
-   w_error = SRT_REJ_IPE;
+    w_error = SRT_REJ_IPE;
 
-   // Can't manage this error through an exception because this is
-   // running in the listener loop.
-   CUDTSocket* ls = locateSocket(listen);
-   if (!ls)
-   {
-       LOGC(mglog.Error, log << "IPE: newConnection by listener socket id=" << listen << " which DOES NOT EXIST.");
-       return -1;
-   }
+    // Can't manage this error through an exception because this is
+    // running in the listener loop.
+    CUDTSocket* ls = locateSocket(listen);
+    if (!ls)
+    {
+        LOGC(mglog.Error, log << "IPE: newConnection by listener socket id=" << listen << " which DOES NOT EXIST.");
```

Comment:
Please fix white spacing here.
As you keep telling me regarding my clang format PR, code formatting is something for later.
Simply run replace of 4 whitespaces with 3 whitespaces in the selection (Ctrl+H or `:%s/\%/    /   /g` in vim (?))

Reply:
Right.